content/docs/gke/deploy/deploy-cli.md

```diff
@@ -74,12 +74,13 @@ Follow these steps to deploy Kubeflow:
     export ZONE=<your target zone> #where the deployment will be created
 
     export PROJECT=<your GCP project>
+    # The value of KFAPP must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'kubeflow-test',  or 'kfc-test'
```

Comment:
Please change this:
```
(e.g. 'kubeflow-test',  or 'kfc-test'
```

to this:
```
(for example,  'kubeflow-test',  or 'kfc-test').
```

The reason is that it's best not to use "e.g." as many people don't know the difference between "e.g." and "i.e.". Instead, spell out the full term. Also, there's an unmatched parenthesis in the original comment.